terminus-ui/autocomplete/src/autocomplete.component.spec.ts

```diff
@@ -168,7 +168,14 @@ describe(`TsAutocompleteComponent`, function() {
 
 
     test(`should allow removal with the backspace key`, () => {
-      jest.useFakeTimers();
+      // NOTE: `dispatchKeyboardEvent` suddenly stopped working for this test during the Angular v8 upgrade
+      const event = document.createEvent('KeyboardEvent');
+      event.initEvent('keydown', true, false);
+      Object.defineProperties(event, {
+        keyCode: { get: () => KEYS.BACKSPACE.keyCode },
+        key: { get: () => KEYS.BACKSPACE.code },
+      });
+
```

Comment:
basically reimplemented `dispatchKeyboardEvent` here... wonder whether v8 makes that helper working intermittently.